modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/portlet/action/GetContentDashboardItemInfoMVCResourceCommand.java

```diff
@@ -502,6 +506,15 @@ private boolean _isShareable(
 		}
 
 		try {
+			String className = _portal.getClassName(classNameId);
+
+			// LPS-111037
+
+			if (Objects.equals(FileEntry.class.getName(), className)) {
```

Comment:
Hey @Stephy6 !!

As it hapends with Blogs, we should never add a explicit dependency to files in the Content Dashboard, take a look into the actual code to see how we are mapping betwen FileEntry and DLFileEntry, for example when we use Search in the Content Dashboard